decision-making-processes.md

```diff
@@ -1,35 +1,157 @@
 
 ## Step 2. Inviting a Conversation around your Suggestion to Find Consensus
 You will then need to get a W3C account and join the W3C Solid Community Group to raise a conversation about your suggestion as an item on the weekly call agenda. By adding the item to the agenda all members of the W3C Solid Community Group will be automatically notified and therefore are able to comment on the pull request or issue directly. 
 
+Anyone can suggest routes forward to the suggested changes with pros and cons of each route forward on the original pull request or issues. Below is a table format where all the final suggested routes forward, pros and cons should be included. 
+
+| Route Forward | Pros to Consider | Cons to Consider |
+| ------------- | ------------- | ------------- |
+| (insert suggestion)  | (insert suggestion)  | (insert suggestion)  |
+
+This step is to make sure that everyone has a chance to voice their thoughts on all the options and their opinions of each so that they feel that all the options are given a chance.
+
 ## Step 3. Coming to a Conclusion through Compromise  
-If there is a difference of opinion, parties are encouraged to talk to find a compromise. The Solid Specification Repository Manager is responsible for processing the suggestion to a change to the Solid specifications and deciding on the route forward. The repository manager of the repository to which the suggestion pertains to will be responsible for merging and closing the pull request or issue.  If a compromise cannot be met the Solid Leader will make the final judgement.
+The Solid Team collects the options, pros, and cons, and then make sure that everyone on the Solid Panel receives a complete copy of all the information via the github pull request and issue.
+
+Each individual in the Panel has one vote even if they have multiple criteria for being in the Solid panel, so each person in the Panel has an equal voice. Each individual can vote on a first and second choice, or they can abstain. Each individual in the Panel has seven days to vote. If the individaul does not vote after seven days their vote will automatically be counted as an abstaining vote. After counting the first choice votes, the votes from the bottom three options get transferred to the second choice vote to avoid strategic voting that does not reflect the true opinion of the Panel. The votes are recounted and the majority vote wins. 
+
+The Solid Team's responsibilities are to:  
+* make sure everyone on the Panel is aware of the vote
+* explain how they can vote
+* collect the final results
+* communicate the outcome
+
+Solid specification 0.9 will be released on July 1st, and new versions of the Solid specification will be released every six months For a change to the Solid specification to be merged there needs to be at least one working implemetation that adheres to the changed Solid specification.
```

Comment:
Let's not bind to a specific timeframe; it should be done when it's good.

Reply:
What defines the spec being good and who is the judge of goodness?

Reply:
> What defines the spec being good

Correctness, clarity, and non-ambiguity.

> and who is the judge of goodness?

The editors are expected to deliver blurbs of text that conform to the above criteria. They deliver these through PRs, which members of the public can discuss on for at least a week.